target/consensource
# Misc UI Updates

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Misc. UI fixes, including:

- Labels for factory profile info
- Clickable call-to-action buttons
- Scaling font size on the landing page
- Rename `authService` -> `AuthService` throughout app for consistency

## Types of changes

What types of changes is this pull request introducing to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `docker-compose up`
- `cd client`
- `npm run build-dev`
- View the described changes and confirm the intended functionality.
